src/answers-headless.ts

```diff
@@ -295,5 +296,14 @@ export default class AnswersHeadless {
     };
     this.stateManager.dispatchEvent('filters/toggleFacetOption', payload);
   }
+
+  toggleFilterOption(filterCollectionId: string, filter: Filter, shouldSelect: boolean): void {
```

Comment:
Is there a reason for having `Filter` and `shouldSelect` rather than allowing a `SelectableFilter` to be used here? It's part of the interface for SelectableFilter and if we used it here we would have more consistent interfaces. 

Side note, if we were to make this interface `toggleFilterOption(filter: SelectableFilter, filterCollectionId)`, then we could easily make `filterCollectionId` optional in the future

Reply:
ah yeah, forgot to SelectableFilter. As for the the order of params, I don't think we would have an unknown/optional `filterCollectionId` scenario right now, but I moved it to last argument anyway just in case we do, like you said